omnibus/package-scripts/publish_image.sh

```diff
@@ -13,7 +13,9 @@ echo $ARTIFACT_PATH
 cp $ARTIFACT_PATH/*.deb Dockerfiles/agent
 
 docker build -t stackstate/${IMAGE_REPO}:${IMAGE_TAG} Dockerfiles/agent
-docker tag stackstate/${IMAGE_REPO}:${IMAGE_TAG} stackstate/${IMAGE_REPO}:latest
+docker tag stackstate/${IMAGE_REPO}:${IMAGE_TAG} 
```

Comment:
There are several problems here:

1) this `tag` command won't work (check `docker tag`) because as it was before needs 2 arguments not one `Usage:  docker tag SOURCE_IMAGE[:TAG] TARGET_IMAGE[:TAG]`

2) we don't want to push `latest` always, but as the ticket https://stackstate.atlassian.net/browse/STAC-5310 says `we are not pushing the latest tag for releases`; this script in fact is used for internal test and for official releases